src/scripts/main.js

```diff
@@ -1 +1,83 @@
 'use strict';
+
+function createFirstPromise() {
+  return new Promise((resolve, reject) => {
+    let isClicked = false;
+
+    document.addEventListener('click', () => {
+      isClicked = true;
+      resolve('First promise was resolved');
+    });
+
+    setTimeout(() => {
+      if (!isClicked) {
+        reject(new Error('First promise was rejected'));
+      }
+    }, 3000);
+  });
+}
+
+function createSecondPromise() {
+  return new Promise((resolve, reject) => {
+    document.addEventListener('click', () => {
+      resolve('Second promise was resolved');
+    });
```

Comment:
Adding multiple event listeners for the same event type ('click') can lead to unexpected behavior if the event is triggered multiple times. Consider using a single event listener or removing the listener after it resolves the promise.